spec/operations/organizations/update_spec.rb

```diff
@@ -18,7 +18,7 @@
 
   let(:operation) { described_class.with(api_user: api_user, id: organization.id.to_s) }
 
-  it 'sets the urls, splitting out translatable content' do
+  it 'sets the content' do
```

Comment:
not sure the new spec name makes more sense. The labels should be generic, translatable names that can be replaced with translations.

Reply:
This is checking that it's setting the untranslated string (`0.label`), not the translated one. Checking that it was properly serializing the translation is what the broken `#urls` spec was supposed to be doing before this PR fixes.

So you're right, it's not accurate since the URLs field it's confirming was updated isn't in contents. But it was also wrong before, I'll update it with something else.